docs/_docs/monitoring-metrics/system-views.adoc

```diff
@@ -168,6 +168,35 @@ This view exposes information about currently running compute tasks started by a
 |USER_VERSION | string |  Task user version
 |===
 
+== JOBS
+
+This view exposes information about currently running compute job started on a node.
+In the Ignite, Job defines part(step) of the compute task
+
+[{table_opts}]
+|===
+|NAME | TYPE |    DESCRIPTION
+|ID | UUID | Job ID
+|SESSION_ID | UUID | Session ID. Note, `SESSION_ID` equal to `TASKS.SESSION_ID` for the jobs that belongs to the specific task
+|ORIGIN_NODE_ID | UUID | Originating node id
+|TASK_NAME | string | Name of the task
+|TASK_CLASSNAME | string | Class name of the task
+|AFFINITY_CACHE_IDS | string | Affinity cache ids
```

Comment:
Can we clarify the purpose of these AFFINITY parameters? Do the parameters list all caches and partitions the job is mapped to? What will be reported in these parameters if the job is broadcasted?

Reply:
`affinityCacheIds` equal to the `cacheName` or `cacheNames` parameter of the `IgniteCompute#affinityCall[Async]` methods.

Reply:
@nizhikov,

Will the `AFFINITY_CACHE/PARTITION_IDS` parameters be empty if no caches/partitions were specified (basically, if you don't use affinityRuns/Calls? Pull the latest changes, check how I altered the description. Is it accurate?

Also, we have this sentence at the top of the JOB's table - Note, `SESSION_ID` is equal to `TASKS.SESSION_ID` for the jobs belonging to a specific task. However, I couldn't find the description of the `TASKS.SESSION_ID`. Could you update the TASK's table?

Reply:
>  parameters be empty if no caches/partitions were specified

Yes.

Reply:
> Could you update the TASK's table?

Done.